styles.css

```diff
@@ -81,10 +93,260 @@
   justify-content: space-between;
 }
 
-.links {
+.social {
+  display: inline;
+}
+
+.headline2 {
+  min-height: calc(100vh - 90px);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.btn {
+  font-family: 'Inter', sans-serif;
+  height: 40px;
+  font-size: 15px;
+  padding: 8px;
+  background: white;
+  border: 1px solid #8993a4;
+}
+
+.color {
+  background: #ff6b00;
+  color: #fff;
+  border: 1px solid #ff6b00;
+}
+
+.container1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+hr {
+  width: 327px;
+  height: 0;
+  border: 1px solid #212121;
+  margin-bottom: 4rem;
+}
+
+.title {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+}
+
+.flex {
+  display: flex;
+  flex-direction: column;
+  align-content: space-between;
+}
+
+.flex h1 {
+  margin-bottom: 1.5rem;
+}
+
+.post1 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+}
+
+.text {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.lesson {
+  display: flex;
+  margin: 0;
+  flex-wrap: wrap;
+  gap: 12px;
+}
+
+.contents {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  gap: 12px;
+}
+
+.post {
+  background-image: url("images/back2.png");
```

Comment:
- [ ] You have done a good job adding your images as background images, but as seen in the screenshot below, your Works section cards images and the About section background images are not rendering on the page. Please confirm whether you have all your images in the Images folder(it seems to have only 1 image) and whether you have the right paths to the images, and make sure they render correctly on the page.

-   ![muma1](https://user-images.githubusercontent.com/87394385/213187260-956b5528-2987-43f7-afcb-47d3b9d19e01.JPG)

-   ![muma2](https://user-images.githubusercontent.com/87394385/213187325-1cfeab14-b0e5-49ff-975c-8804a6b36d9e.JPG)